Dockerfile

```diff
@@ -2,13 +2,23 @@ FROM centos:7
 MAINTAINER Federico Simoncelli <fsimonce@redhat.com>
 
 RUN yum update -y && \
-    yum install -y golang openscap-scanner && \
+    yum install -y golang openscap-scanner git && \
     yum clean all
 
-WORKDIR /var/lib/image-inspector
+RUN GOBIN=/usr/bin \
+    GOPATH=/go \
+    go get github.com/rancher/trash
 
 COPY .  /go/src/github.com/openshift/image-inspector
 
+WORKDIR /go/src/github.com/openshift/image-inspector/
+
+RUN GOBIN=/usr/bin \
+    GOPATH=/go \
+    trash
```

Comment:
@ilackarms what is this needed for? is it downloading the missing dependencies? Looking at skopeo I couldn't find anything similar. When are they running it?